io_scene_godot/export_godot.py

```diff
@@ -136,7 +137,7 @@ def export_object(self, obj, parent_gd_node):
         if (exported_node.parent is not None and
                 exported_node.parent.get_type() == 'CollisionShape'):
             exported_node['transform'] = (
-                mathutils.Matrix.Rotation(math.radians(90), 4, 'X') @
```

Comment:
This is not the same rotation (-90 vs 90),  a little hacky here.

Reply:
I changed this because its easier to interpret and it confused me at first, _AXIS_CORRECT is applied to the CollisionShape, so it makes sense _AXIS_CORRECT.invert() would be applied to the child.

Reply:
Ah, I didn't notice the `invert`. Sorry, it's my fault. This is definitely a good change